challenge1.exs

```diff
@@ -1,3 +1,65 @@
 defmodule Challenge1 do
+  def divisible3(a, b) do
+    inicio = inicio(a, 0, "")
```

Comment:
Hi @Santiorozco7 please try to rename all your variable names, documentation, etc in English :)